fs/bcachefs/bcachefs_format.h

```diff
@@ -1483,7 +1493,9 @@ static inline _Bool bch2_csum_type_is_encryption(enum bch_csum_type type)
 #define BCH_CSUM_OPTS()			\
 	x(none,			0)	\
 	x(crc32c,		1)	\
-	x(crc64,		2)
+	x(crc64,		2)	\
+	x(xxhash,		3)	\
```

Comment:
you are right.  The idea was "if someone makes a bcachefs filesystem with xxhash, and then builds a kernel without xxhash support, it may be useful if the kernel can print in dmesg which algorithm isn't supported". But no idea how I should do this :s

Reply:
I updated the commit so these options are no longer included.  I'll further think of a strategy to handle unsupported checksum algorithms